src/features/aimode/MozAIModePage.ts

```diff
@@ -35,6 +72,15 @@ class MozAIModePage extends LitElement {
     this.closeSidebar()
   }
 
+  protected firstUpdated(_changedProperties: PropertyValues): void {
+    const mozMentionInput = document.querySelector('moz-mention-input')
+    mozMentionInput?.addEventListener('mention-input:filter', (e: any) => {
+      // This is where you would handle the filter event and then we can update the options given to the
+      // mention input component and not have to worry about filerting in the component itself.
+      console.log('Received mention-input:filter event:', e.detail)
```

Comment:
don't need to implement now but what's the expected way to filter? modify mockOptions?